modules/cnv-about-node-network-state.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-node-network-state.adoc
+
+[id="cnv-about-node-network-state_{context}"]
+= About node network state
+
+{CNVProductNameStart} uses `nmstate` to report on and configure the state of the node network. This can be applied to a single node, a subset of nodes, or all of the nodes in the cluster. This makes it possible to modify network policy configuration, such as by creating a linux bridge on all nodes, by applying a single configuration manifest to the cluster.
```

Comment:
The mention of single/subset/all nodes here feels odd. You have not introduced the Policy yet and by reading this it sounds like the network reporting is done for a subset (which is not true, we always report state of all nodes).

Reply:
I've removed that sentence now.